test/conformance/enqueue/enqueue_adapter_cuda.match

```diff
@@ -1,10 +1,13 @@
+{{NONDETERMINISTIC}}
 urEnqueueKernelLaunchTest.InvalidKernelArgs/*
 urEnqueueKernelLaunchKernelWgSizeTest.NonMatchingLocalSize/*
 urEnqueueKernelLaunchKernelSubGroupTest.Success/*
 urEnqueueMemBufferMapTestWithWriteFlagParam.SuccessWrite/*__UR_MAP_FLAG_WRITE_INVALIDATE_REGION
 urEnqueueUSMAdviseWithParamTest.Success/*__UR_USM_ADVICE_FLAG_DEFAULT
 urEnqueueUSMAdviseTest.MultipleParamsSuccess/*
-urEnqueueUSMPrefetchWithParamTest.Success/*__UR_USM_MIGRATION_FLAG_DEFAULT
-urEnqueueUSMPrefetchWithParamTest.CheckWaitEvent/*__UR_USM_MIGRATION_FLAG_DEFAULT
+urEnqueueUSMPrefetchWithParamTest.Success/*__UR_USM_MIGRATION_FLAG_HOST_TO_DEVICE
+urEnqueueUSMPrefetchWithParamTest.Success/*__UR_USM_MIGRATION_FLAG_DEVICE_TO_HOST
+urEnqueueUSMPrefetchWithParamTest.CheckWaitEvent/*__UR_USM_MIGRATION_FLAG_HOST_TO_DEVICE
+urEnqueueUSMPrefetchWithParamTest.CheckWaitEvent/*__UR_USM_MIGRATION_FLAG_DEVICE_TO_HOST
 urEnqueueTimestampRecordingExpTest.Success/*
-urEnqueueTimestampRecordingExpTest.SuccessBlocking/*
+urEnqueueTimestampRecordingExpTest.SuccessBlocking/*
```

Comment:
We've recently merged #2479 which removes uses of match files from the CTS, the new mechanism for tracking know failures is described [here](https://github.com/oneapi-src/unified-runtime/blob/main/test/conformance/README.md#known-failures).